pages/templates/pages/microsite_page.html

```diff
@@ -1,12 +1,210 @@
 {% extends "pages/base_page.html" %}
 
-{% load wagtailcore_tags %}
+{% load wagtailcore_tags wagtailimages_tags %}
+
+{% block meta %}
+  {{ block.super }}
+  <style>
+    #custom_header {
+      {% if page.custom_header_background_color %}
+        background-color: #{{ page.custom_header_background_color }};
+      {% else %}
+        background: transparent;
+      {% endif %}
+
+      {% if page.show_accent_border %}
+        border-top: solid 8px #{{page.accent_border_color}};
+      {% endif %}
+    }
+
+    #primary-cta {
+      {% if page.button_background_color %}
+        background-color: #{{page.button_background_color}};
+        border-color: #{{page.button_background_color}};
+      {% endif %}
+
+      {% if page.button_text_color %}
+        color: #{{page.button_text_color}};
+      {% endif %}
+    }
+
+    .microsite-content a#primary-cta {
```

Comment:
Why styling `#primary-cta` twice? Also don't need selector like this for id tag?

Reply:
Need this because `#primary_content a` was overriding this style.

Reply:
In this case I suggest to use `!important` instead, or at least add code comments. This looks strange and could easily be changed by accident.